pkg/morph/client/notary.go

```diff
@@ -427,9 +474,15 @@ func (c *Client) notaryInvoke(committee, invokedByAlpha bool, contract util.Uint
 		return err
 	}
 
-	until, err := c.notaryTxValidationLimit()
-	if err != nil {
-		return err
+	var until uint32
+
+	if vub != nil {
+		until = *vub
+	} else {
+		until, err = c.notaryTxValidationLimit()
```

Comment:
I would rather keep `vub` as a `uint32` type and use `c.notaryTxValidationLimit()` at `NotaryInvokeNotAlpha()` function. Are there any other cases with nil `vub`?